.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+- node
+script:
+  - npm test -- --coverage
+after_script:
+  - COVERALLS_REPO_TOKEN=$coveralls_repo_token npm run coveralls
```

Comment:
I was wondering what line 7 does? Is it to show that table for test coverage?

Reply:
@phucciTr Yeah! This sends the code coverage over to Coveralls' API, then it shows up on the README.md badge. Super clean and easy to use!